CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Add: entityNameExp to Configuration Model to allow define device.name using expressions
```

Comment:
No issue associated?

Reply:
It is related with a fix in the IOTA-Lib (see PR https://github.com/telefonicaid/iotagent-node-lib/pull/1255), but without issue in the IOTA Manager repo itself.

NTC